mix/CodeModel.cpp

```diff
@@ -135,8 +135,8 @@ CompiledContract::CompiledContract(const dev::solidity::CompilerStack& _compiler
 
 	CollectDeclarationsVisitor visitor(&m_functions, &m_locals, &m_storage);
 	contractDefinition.accept(visitor);
-	m_assemblyItems = filterLocations(_compiler.getRuntimeAssemblyItems(name), contractDefinition, m_functions);
-	m_constructorAssemblyItems = filterLocations(_compiler.getAssemblyItems(name), contractDefinition, m_functions);
+	m_assemblyItems = filterLocations(*_compiler.getRuntimeAssemblyItems(name), contractDefinition, m_functions);
```

Comment:
@arkpar There is a minor, tiny touch to Mix's codebase here so I think it's prudent to also notify you.